projects/fourier/lib/fourier/utilities/system.rb

```diff
@@ -9,12 +9,16 @@ def self.system(*args)
         Kernel.system(*args) || exit(1)
       end
 
-      def self.tuist(*args)
-        Dir.chdir(Constants::TUIST_DIRECTORY) do
-          self.system("swift", "build")
-          @tuist = ".build/debug/tuist"
-          self.system(@tuist, *args)
+      def self.tuist(*args, **kwargs)
+        if kwargs[:from_source]
+          Dir.chdir(Constants::TUIST_DIRECTORY) do
+            self.system("swift", "build")
+            @tuist = ".build/debug/tuist"
+          end
+        else
+          @tuist = "/usr/local/bin/tuist"
```

Comment:
is this the right path for M1 macs?

Reply:
Nope we are using /opt for some on M1. I'll update this to drop the path prefix, imagine it'll select whatever is in the users path.

Reply:
Actually we use `/usr/local/bin` instead of the brew approach of supporting two separate binaries. Tuist is a universal binary

Reply:
thx for the information, just wanted to make sure this work's on M1 as we don't have M1 CI to test before...